src/server/pkg/worker/api_server.go

```diff
@@ -406,9 +402,7 @@ func (a *APIServer) Process(ctx context.Context, req *ProcessRequest) (resp *Pro
 	if err := os.MkdirAll(client.PPSOutputPath, 0666); err != nil {
 		return nil, err
 	}
-	logger.Logf("beginning to process user input")
 	userlog, err := a.runUserCode(ctx, logger)
-	logger.Logf("finished processing user input")
```

Comment:
So I changed my mind about these log statements: they actually proved really helpful when I was debugging GF's pipelines.

What do you think about this:

* Keep these log statements, but prepend them with something like `PACH: `.
* In `GetLogs`, filter out the lines that contain `PACH: ` if `raw` is false.

That is, normally we don't see these lines (assuming `raw` defaults to true; see the comment below).  When we really need to debug something, we turn off `raw` and get these lines.